src/client/pps/pps.proto

```diff
@@ -109,29 +109,17 @@ message DeletePipelineRequest {
   Pipeline pipeline = 1;
 }
 
-service API {
-  rpc CreateJob(CreateJobRequest) returns (Job) {}
-  rpc InspectJob(InspectJobRequest) returns (JobInfo) {}
-  rpc ListJob(ListJobRequest) returns (JobInfos) {}
-  rpc GetLogs(GetLogsRequest) returns (stream google.protobuf.BytesValue) {}
-
-  rpc CreatePipeline(CreatePipelineRequest) returns (google.protobuf.Empty) {}
-  rpc InspectPipeline(InspectPipelineRequest) returns (PipelineInfo) {}
-  rpc ListPipeline(ListPipelineRequest) returns (PipelineInfos) {}
-  rpc DeletePipeline(DeletePipelineRequest) returns (google.protobuf.Empty) {}
-
+message SubscribeNewPipelineRequest {
```

Comment:
I'm a bit confused about why this proto is in this file, is it used in one of the API calls defined here?